app/models/presenters/plate_presenter.rb

```diff
@@ -151,6 +151,10 @@ def qc_thresholds
 
     private
 
+    def skip_validation_for_single_pcr_cycle_for_all_wells?
+      false
```

Comment:
This looks like it's for future use or perhaps unimplemented?  Should it be removed or have a comment added to it?

Reply:
Nope that explains it.  Maybe you could just add a comment to this so that it's clear why this is always `false` here.  Normally these booleans are based on properties of the class, but in this case you're just overriding base class behaviour.